schemas/notes.py

```diff
@@ -1,10 +1,16 @@
+from ma import ma
 from models.user import UserModel
 from models.tickets import TicketModel
+from models.notes import NotesModel
 from utils.time import time_format
-from marshmallow import fields, Schema, validates, ValidationError
+from marshmallow import fields, validates, ValidationError, post_load, EXCLUDE
 
 
-class NotesSchema(Schema):
+class NotesSchema(ma.SQLAlchemyAutoSchema):
+    class Meta:
+        model = NotesModel
+        unknown = EXCLUDE
```

Comment:
I don't think this is necessary. Is there a reason for adding `unknown=EXCLUDE` ?